app/pages/resource/reservation-form/ReservationFormContainer.js

```diff
@@ -14,11 +15,22 @@ const reservationFormSelector = createSelector(
   })
 );
 
+const actions = {
+  makeReservation,
+};
+
 function mergeProps(stateProps, dispatchProps, ownProps) {
-  return Object.assign({}, ownProps, stateProps, dispatchProps, {
-    onSubmit: () => {},
-  });
+  const props = { ...ownProps, ...stateProps, ...dispatchProps };
+  return {
+    ...props,
+    onSubmit: values => props.makeReservation({
```

Comment:
- [ ] A test for the onSubmit function could be nice since it combines values from different sources.